src/add_metadata.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Adding extra metadata to the tests.
+ * 
+ * Note that this module is _not_ part of the report generation; it is supposed to be ran separately. It is kept along side the 
+ * report generation script because it reuses some tools in the library.
+ * 
+ * Note that the generation of an EPUB file (ie, the zipped content) is done in a separate module in the library.
+ * 
+ * 
+ *  @packageDocumentation
+ */
+
+import * as fs_old_school from "fs";
+const fs = fs_old_school.promises;
+
+import {get_list_dir, isDirectory} from './lib/data';
+import { Constants } from './lib/types';
+import { create_epub } from './lib/epub';
+
+/**
+ * The extra metadata items to be added to the package metadata
+ */
+const new_metadata: string[] = [
```

Comment:
Nit: ideally I think this would be a flag that you pass, since these elements will have to be removed the next time we want to use this script. But that's a change we can make next time :)

Reply:
While I was at it, I made the change in the code. If ever we have to reuse this code, other changes may be necessary, but better get this out of the way:-)